llvm/include/llvm/Transforms/IPO/SampleProfileMatcher.h

```diff
@@ -19,6 +19,9 @@
 
 namespace llvm {
 
+using Anchor = std::pair<LineLocation, FunctionId>;
```

Comment:
nit: instead of defining `Anchor` which is not used by `AnchorMap`, how about `using AnchorList = std::vector<std::pair<LineLocation, FunctionId>>` and not defining a common `Anchor` type?